### DISCUSSION TITLE
From graphql 15.5.0 schema.json is significantly larger

### DISCUSSION THREAD

**Discussion by frederikhors** [urql-graphql/urql#1347] [Ideas]
I'm using [codegen with introspection plugin](https://graphql-code-generator.com/docs/plugins/introspection) to genereate it from *.graphql files and your code [here](https://formidable.com/open-source/urql/docs/graphcache/schema-awareness/#optimizing-a-schema) to minify my GraphQL schema.json.

After the [graphql upgrade from 15.4.0 to 15.5.0](https://github.com/graphql/graphql-js/releases/tag/v15.5.0) in my schema.json there are hundreds of new `"isDeprecated":false,"deprecationReason":null"`.

1. Can we do something here?
1. Do we need these in schema? Maybe only when `isDeprecated` is `true`?

This is the PR that adds them, I think: https://github.com/graphql/graphql-js/pull/2855.

**Comment by JoviDeCroock** [comment]
Odd, according to the docs it should be false by default https://github.com/graphql/graphql-js/pull/2855/files#diff-af06b2319a79ea80ff786c69e9a529e7045642fa8bce4a10cda2ee1d37f15bfbR24 might be a bug on their side?

We aren't removing this property for normal fields either, we only prune the regulars during minify.

**Reply by frederikhors** [reply]
I do not understand. Should I open an issue there?

**Comment by kitten** [comment]
The output of our utility is unchanged at least on an API that hasn't changed and only results in these fields on `__EnumValue` and `__Field` which is expected. Could there be a discrepancy in how you're generating the schema output on your end? This is for instance how we test it: https://github.com/JoviDeCroock/threed-web/blob/07472bfeb160dc4d6f8843d6a0e559ba80ca70b1/scripts/schema.js#L9-L19

I just upgraded this to the latest version of `graphql` (both on the client-side and server-side) and am not seeing any changes in the output.
